doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -551,21 +547,20 @@ pass additional arguments such as ``using relation``.
 .. tacv:: setoid_replace @term with @term [in @ident] [using relation @term] [by @tactic]
```

Comment:
The optional parts should normally be enclosed in `{?` and `}` instead of `[` and `]`.